pyaraucaria/airmass_calculator.py

```diff
@@ -0,0 +1,11 @@
+import math
+
+class AirmassCalculator:
+    @staticmethod
+    def calculate_airmass(elevation: float) -> float:
+        if not 0 <= elevation <= 90:
```

Comment:
Add at least short _docstring_, let the users know what are the units of parameter (degrees), give a source to the method you are using